File: Java/UberBlack.java
import java.util.Map;

class UberBlack extends Car{
    Map<String, Map<String,Integer>> typeCarAccepted;
    Map<String> seatMaterial;



    public UberBlack(String license, Account driver, Map<String, Map<String,Integer>> typeCarAccepted){
        super(licence, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    
    }
}

File: Python/UberBlack.py
from Car import Car

class UberBlack(Car):
    TypeCarAccepted =[]
    SeatMaterial = []
    
    def __init__(self,license,driver,TypeCarAccepted,SeatMaterial):
        super().__init__(license,driver)
        self.TypeCarAccepted=TypeCarAccepted
        self.SeatMaterial = SeatMaterial